.github/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:testing
+FROM debian:sid
```

Comment:
yeah, I know testing is busted today. I switched to it because unstable would be unusable sometimes. testing should be fixed in a couple of days once gcc-14 migrates.

Reply:
> yeah, I know testing is busted today. I switched to it because unstable would be unusable sometimes. testing should be fixed in a couple of days once gcc-14 migrates.

My personal experience is sid tends to be more stable, I'll drop this commit.

Reply:
yeah, testing is having a rough time right now with gcc-14 taking a couple of months to migrate due to testing regressions. I guess if it doesn't recover soon, we're gonna need to switch to unstable or not have CI at all.